### ISSUE TITLE
NPE after upgrading to Maven plugin 0.14.5

### ISSUE THREAD

**Issue by uhafner** [revapi/revapi#253]
After upgrading to the Maven plugin 0.14.5 I get an NPE:
```
Caused by: java.lang.NullPointerException
    at org.revapi.java.checks.common.SerializationChecker.computeStructuralId (SerializationChecker.java:318)
    at org.revapi.java.checks.common.SerializationChecker.doEnd (SerializationChecker.java:216)
    at org.revapi.java.spi.CheckBase.visitEnd (CheckBase.java:243)
    at org.revapi.java.JavaElementDifferenceAnalyzer.endAnalysis (JavaElementDifferenceAnalyzer.java:254)
    at org.revapi.java.JavaElementDifferenceAnalyzer.endAnalysis (JavaElementDifferenceAnalyzer.java:70)
    at org.revapi.Revapi.analyze (Revapi.java:454)
    at org.revapi.Revapi.analyze (Revapi.java:445)
    at org.revapi.Revapi.analyzeWith (Revapi.java:341)
    at org.revapi.Revapi.analyze (Revapi.java:212)
    at org.revapi.maven.Analyzer.analyze (Analyzer.java:445)
    at org.revapi.maven.AbstractRevapiMojo.analyze (AbstractRevapiMojo.java:416)
    at org.revapi.maven.CheckMojo.doExecute (CheckMojo.java:97)
    at org.revapi.maven.AbstractRevapiMojo.execute (AbstractRevapiMojo.java:397)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:972)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)

```

I still have some warnings in my project setup, maybe that is the reason:
```
[WARNING] Extension 'versionFormat' doesn't declare a JSON schema but XML contains its configuration. Cannot convert it into JSON and will ignore it!
[WARNING] Extension 'failBuildOnProblemsFound' doesn't declare a JSON schema but XML contains its configuration. Cannot convert it into JSON and will ignore it!
[WARNING] The `revapi.ignore` extension is deprecated. Consider using the `revapi.differences` instead.
[WARNING] revapi.semver.ignore transform has been deprecated. Please use the new revapi.versions which has better integration with the new Revapi features.
```

Here is the whole build result: https://github.com/jenkinsci/warnings-ng-plugin/runs/3914658212?check_suite_focus=true

And here my [configuration in the pom.xml](https://github.com/jenkinsci/warnings-ng-plugin/blob/7304fa52163084845d87d5bceafe37777f2eba22/plugin/pom.xml#L645).

**Comment by metlos** [comment]
Should be fixed by c4ad7100a94c864d1bc5309cd69aec7364b782ed

**Comment by metlos** [comment]
Fixed in `revapi-java:0.26.1`. Thanks for reporing this.
